File: Laba5/OP2,5Python/OP2,5Python/rectangle.py
import math
from TQuadrangle import TQuadrangle
class Rectangle(TQuadrangle):
    def __init__(self, Coords, typ = "Rectangle"):
        super().__init__(Coords, typ)
        self.area = self.countArea()
    def countArea(self):
        Side1 = math.sqrt(pow(self.Coords[1].x - self.Coords[0].x, 2) + pow(self.Coords[1].y - self.Coords[0].y, 2))
        Side2 = math.sqrt(pow(self.Coords[2].x - self.Coords[1].x, 2) + pow(self.Coords[2].y - self.Coords[1].y, 2))
        return Side1 * Side2

File: Laba5/Laba2.5/Laba2.5/rectangle.cpp
#include "rectangle.h"

double rectangle::countArea() { // Обрахунок площі прямокутника
	double Side1, Side2;
	Side1 = sqrt(pow(V[1].x - V[0].x, 2) + pow(V[1].y - V[0].y, 2)); // Довжина першої сторони
	Side2 = sqrt(pow(V[2].x - V[1].x, 2) + pow(V[2].y - V[1].y, 2)); // Довжина другої сторони
	return Side1 * Side2;
}
